common/db_store/db.py

```diff
@@ -0,0 +1,84 @@
+from enum import Enum
+import pymongo
+from abc import ABC, abstractmethod
+from common.db_store.error import DBError
+
+
+class DB(ABC):
+    @abstractmethod
+    def get_data(self, collection, **kwargs):
+        pass
+
+    @abstractmethod
+    def save_data(self, collection, data, **kwargs):
+        pass
+
+
+class STATUSES(Enum):
+    SUCCESS = "success"
+
+
+class MongoDB(DB):
+
+    def __init__(self, endpoint, db_name, **kwargs) -> None:
+        """
+        Initialize DB connection.
+        Args:
+            endpoint (str): MongoDB server endpoints.
+                Ex. mongodb://mongo-0.mongo,mongo-1.mongo,mongo-2.mongo:27017
+            db_name (str): Name of Database.
+        """
+        # TODO : Maintain maximum 10 connection in connection pool
+        try:
+            self._client = pymongo.MongoClient(endpoint)
+            self._db = self._client[db_name]
+        except Exception as e:
+            # Log.error(f"Unable to connect database server endpoints \
```

Comment:
remove if not required

Reply:
Currently in cortx-halo logging is not enabled, & We're planning to enable it in near future, So it will be good to have these comments.

Reply:
could you add a TODO statement with it, regarding the same